deploy/chart/values.yaml

```
@@ -54,7 +52,7 @@ deploy:
   # and the storage-controller will handle the cleanup of the subpath in NAS filesystem.
   withStorageController: false
 
-  newProvisionerToken: true
+  ramTokenV2: true
```

```suggestion
  ramToken: v2
```
if we ever have a v3 in the future, we want to prevent user from enable v2 and v3 simultaneously.